tlatools/org.lamport.tlatools/src/pcal/PCalTLAGenerator.java

```diff
@@ -45,9 +45,6 @@ public void removeNameConflicts() throws RemoveNameConflictsException
         }
 
         st.Disambiguate();
-        if (st.disambiguateReport.size() > 0)
```

Comment:
I'm not sure we should roll this change into this PR for backward compatibility reasons.  Is there a reason for removal besides it being annoying?

Reply:
No other reason. I'll revert it then.